src/chesster.js

```diff
@@ -547,6 +547,7 @@ function(bot,message) {
 /* mods */
 
 leagueResponse(['summon mods'], 'formatSummonModsResponse');
+leagueResponse(['notify mods'], 'formatSummonModsResponse');
```

Comment:
I believe the reason why that first parameter is a list is that you can pass in multiple regexes there.